lib/generators/format/templates/format.html.erb

```diff
@@ -6,4 +6,4 @@
   </div>
 </div>
 
-<%%= render 'components/lead-paragraph', text: @content_item.description %>
+<%= render 'govuk_component/lead_paragraph', text: @content_item.description %>
```

Comment:
This should still be `<%%` – it's a peculiar style used to escape a percent in ERB.
eg https://stackoverflow.com/questions/2322413/how-do-i-escape-erb-code-in-fixtures

Reply:
Fixed.